scripts/build_utils.sh

```diff
@@ -1453,6 +1453,10 @@ setup_rpm_build_deps() {
             $SUDO yum-config-manager --enable centos-sclo-rh-testing
         fi
     elif [ "$RELEASE" = 8 ]; then
+        # for grpc-devel
+        # See https://copr.fedorainfracloud.org/coprs/ceph/grpc/
+        $SUDO yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
@dmick is EPEL 8 already enabled on the builders by default?

Reply:
@ktdreyer , if already installed, this command would exit with code 0
```shell
[root@1937af58a4d5 /]# yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
Failed to set locale, defaulting to C.UTF-8
Extra Packages for Enterprise Linux 8 - x86_64                                                                  3.4 MB/s |  16 MB     00:04
Last metadata expiration check: 0:00:05 ago on Fri Feb 23 16:30:36 2024.
epel-release-latest-8.noarch.rpm                                                                                4.4 kB/s |  25 kB     00:05
Package epel-release-8-19.el8.noarch is already installed.
Dependencies resolved.
Nothing to do.
Complete!
[root@1937af58a4d5 /]# echo $?
0
```

I'd aim to minimize assumptions about runners.
wdyt?

Reply:
I think it will be simpler, faster and easier to maintain if you drop this addition.

Reply:
I don't see any harm in making sure epel8 is available at the place where it must be.  If it's preinstalled on the builders, it's a long way away from here, and it's good to keep like operations together just for documentation reasons if nothing else.

Reply:
Ok, thank you.